### PR TITLE
Don't include missing tokens when creating a breakpoint span.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @rchande

File: src/EditorFeatures/CSharpTest/EditAndContinue/BreakpointSpansTests.cs [hunk 1]
```diff
@@ -1188,7 +1188,7 @@ void Goo()
 }");
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.DebuggingBreakpoints)]
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/CSharpTest/EditAndContinue/BreakpointSpansTests.cs]: the trait is on hte class (and it thus inherited by all members inside).  removed these two stragglers.

File: src/EditorFeatures/CSharpTest/EditAndContinue/BreakpointSpansTests.cs [hunk 2]
```diff
@@ -1700,7 +1700,7 @@ void Goo()
     try
     {
     }
-    catch(Exception e) [|if$$ (e.Message != null)|]
+    catch(Exception e) [|when$$ (e.Message != null)|]
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/CSharpTest/EditAndContinue/BreakpointSpansTests.cs]: "if (" as an exception clause was the original syntax.  These tests were not validating actual c# syntax cases.

File: src/EditorFeatures/CSharpTest/EditAndContinue/BreakpointSpansTests.cs [hunk 3]
```diff
@@ -4656,5 +4656,18 @@ static void M()
     }
 }");
         }
+
+        [Fact, WorkItem(98990, "https://developercommunity.visualstudio.com/content/problem/98990")]
```
Comment (rchande) [inline src/EditorFeatures/CSharpTest/EditAndContinue/BreakpointSpansTests.cs]: Broken link?

**Pull request merged**